2016-10-11 10:30:05:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:42:24:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:42:24:No exception in execute_script method.
2016-10-11 10:42:24:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:42:41:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:43:56:No exception in execute_script method.
2016-10-11 10:43:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:43:56:No exception in execute_script method.
2016-10-11 10:43:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:43:56:No exception in execute_script method.
2016-10-11 10:43:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:43:57:No exception in execute_script method.
2016-10-11 10:43:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:43:57:No exception in execute_script method.
2016-10-11 10:43:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:43:57:No exception in execute_script method.
2016-10-11 10:43:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:43:58:No exception in execute_script method.
2016-10-11 10:43:58:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:43:59:No exception in execute_script method.
2016-10-11 10:44:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:44:56:No exception in execute_script method.
2016-10-11 10:44:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:44:57:No exception in execute_script method.
2016-10-11 10:44:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:44:57:No exception in execute_script method.
2016-10-11 10:44:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:44:58:No exception in execute_script method.
2016-10-11 10:44:58:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:44:58:No exception in execute_script method.
2016-10-11 10:44:58:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:44:58:No exception in execute_script method.
2016-10-11 10:44:58:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:45:00:No exception in execute_script method.
2016-10-11 10:45:00:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:45:00:No exception in execute_script method.
2016-10-11 10:45:00:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:45:01:No exception in execute_script method.
2016-10-11 10:45:01:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:45:02:No exception in execute_script method.
2016-10-11 10:46:15:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:46:15:No exception in execute_script method.
2016-10-11 10:46:15:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:46:16:No exception in execute_script method.
2016-10-11 10:46:16:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:46:18:No exception in execute_script method.
2016-10-11 10:46:18:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:46:18:No exception in execute_script method.
2016-10-11 10:46:18:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:46:18:No exception in execute_script method.
2016-10-11 10:46:18:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:46:18:No exception in execute_script method.
2016-10-11 10:46:18:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:46:19:No exception in execute_script method.
2016-10-11 10:46:19:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:46:19:No exception in execute_script method.
2016-10-11 10:46:19:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:46:20:No exception in execute_script method.
2016-10-11 10:46:20:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:46:20:No exception in execute_script method.
2016-10-11 10:46:43:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:46:43:No exception in execute_script method.
2016-10-11 10:46:43:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:46:44:No exception in execute_script method.
2016-10-11 10:46:44:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:46:45:No exception in execute_script method.
2016-10-11 10:46:45:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:46:45:No exception in execute_script method.
2016-10-11 10:46:45:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:46:46:No exception in execute_script method.
2016-10-11 10:46:46:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:46:47:No exception in execute_script method.
2016-10-11 10:46:47:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:46:47:No exception in execute_script method.
2016-10-11 10:46:47:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:46:47:No exception in execute_script method.
2016-10-11 10:46:47:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:46:48:No exception in execute_script method.
2016-10-11 10:46:48:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:46:48:No exception in execute_script method.
2016-10-11 10:51:05:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:51:05:No exception in execute_script method.
2016-10-11 10:51:05:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:51:07:No exception in execute_script method.
2016-10-11 10:51:07:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:51:08:No exception in execute_script method.
2016-10-11 10:51:08:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:51:08:No exception in execute_script method.
2016-10-11 10:51:08:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:51:08:No exception in execute_script method.
2016-10-11 10:51:08:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:51:09:No exception in execute_script method.
2016-10-11 10:51:09:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:51:09:No exception in execute_script method.
2016-10-11 10:51:09:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:51:10:No exception in execute_script method.
2016-10-11 10:51:10:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:51:11:No exception in execute_script method.
2016-10-11 10:51:11:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:51:11:No exception in execute_script method.
2016-10-11 10:51:31:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:51:31:No exception in execute_script method.
2016-10-11 10:51:31:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:51:32:No exception in execute_script method.
2016-10-11 10:51:32:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:51:33:No exception in execute_script method.
2016-10-11 10:51:33:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:51:33:No exception in execute_script method.
2016-10-11 10:51:33:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:51:34:No exception in execute_script method.
2016-10-11 10:51:34:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:51:35:No exception in execute_script method.
2016-10-11 10:51:35:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:51:35:No exception in execute_script method.
2016-10-11 10:51:35:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:51:36:No exception in execute_script method.
2016-10-11 10:51:36:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:51:36:No exception in execute_script method.
2016-10-11 10:51:36:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:51:37:No exception in execute_script method.
2016-10-11 10:53:10:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 10:53:10:No exception in execute_script method.
2016-10-11 10:53:10:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 10:53:12:No exception in execute_script method.
2016-10-11 10:53:12:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 10:53:13:No exception in execute_script method.
2016-10-11 10:53:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 10:53:13:No exception in execute_script method.
2016-10-11 10:53:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 10:53:13:No exception in execute_script method.
2016-10-11 10:53:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 10:53:14:No exception in execute_script method.
2016-10-11 10:53:14:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 10:53:14:No exception in execute_script method.
2016-10-11 10:53:14:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 10:53:15:No exception in execute_script method.
2016-10-11 10:53:15:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 10:53:15:No exception in execute_script method.
2016-10-11 10:53:15:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 10:53:16:No exception in execute_script method.
2016-10-11 11:06:27:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 11:06:28:No exception in execute_script method.
2016-10-11 11:06:28:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 11:06:29:No exception in execute_script method.
2016-10-11 11:06:29:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 11:06:30:No exception in execute_script method.
2016-10-11 11:06:30:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/like_test.py
2016-10-11 11:06:30:No exception in execute_script method.
2016-10-11 11:06:30:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 11:06:30:No exception in execute_script method.
2016-10-11 11:06:30:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 11:06:31:No exception in execute_script method.
2016-10-11 11:06:31:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 11:06:32:No exception in execute_script method.
2016-10-11 11:06:32:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 11:06:32:No exception in execute_script method.
2016-10-11 11:06:32:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 11:06:32:No exception in execute_script method.
2016-10-11 11:06:32:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 11:06:33:No exception in execute_script method.
2016-10-11 11:06:33:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 11:06:33:No exception in execute_script method.
2016-10-11 11:07:19:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 11:07:20:No exception in execute_script method.
2016-10-11 11:07:20:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 11:07:21:No exception in execute_script method.
2016-10-11 11:07:21:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 11:07:22:No exception in execute_script method.
2016-10-11 11:07:22:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/like_test.py
2016-10-11 11:07:23:No exception in execute_script method.
2016-10-11 11:07:23:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 11:07:23:No exception in execute_script method.
2016-10-11 11:07:23:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 11:07:23:No exception in execute_script method.
2016-10-11 11:07:23:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 11:07:24:No exception in execute_script method.
2016-10-11 11:07:24:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 11:07:24:No exception in execute_script method.
2016-10-11 11:07:24:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 11:07:25:No exception in execute_script method.
2016-10-11 11:07:25:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 11:07:25:No exception in execute_script method.
2016-10-11 11:07:25:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 11:07:26:No exception in execute_script method.
2016-10-11 11:08:50:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 11:08:52:No exception in execute_script method.
2016-10-11 11:08:52:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 11:08:53:No exception in execute_script method.
2016-10-11 11:08:53:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 11:08:54:No exception in execute_script method.
2016-10-11 11:08:54:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/like_test.py
2016-10-11 11:08:55:No exception in execute_script method.
2016-10-11 11:08:55:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 11:08:55:No exception in execute_script method.
2016-10-11 11:08:55:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 11:08:55:No exception in execute_script method.
2016-10-11 11:08:55:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 11:08:56:No exception in execute_script method.
2016-10-11 11:08:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 11:08:56:No exception in execute_script method.
2016-10-11 11:08:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 11:08:57:No exception in execute_script method.
2016-10-11 11:08:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 11:08:57:No exception in execute_script method.
2016-10-11 11:08:57:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 11:08:58:No exception in execute_script method.
2016-10-11 11:09:09:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 11:09:09:No exception in execute_script method.
2016-10-11 11:09:09:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 11:09:10:No exception in execute_script method.
2016-10-11 11:09:10:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 11:09:11:No exception in execute_script method.
2016-10-11 11:09:11:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/like_test.py
2016-10-11 11:09:13:No exception in execute_script method.
2016-10-11 11:09:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 11:09:13:No exception in execute_script method.
2016-10-11 11:09:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 11:09:13:No exception in execute_script method.
2016-10-11 11:09:13:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 11:09:14:No exception in execute_script method.
2016-10-11 11:09:14:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 11:09:14:No exception in execute_script method.
2016-10-11 11:09:14:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 11:09:15:No exception in execute_script method.
2016-10-11 11:09:15:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 11:09:16:No exception in execute_script method.
2016-10-11 11:09:16:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 11:09:17:No exception in execute_script method.
2016-10-11 11:09:49:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/batch_follow_test.py
2016-10-11 11:09:49:No exception in execute_script method.
2016-10-11 11:09:49:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/collect_post_test.py
2016-10-11 11:09:51:No exception in execute_script method.
2016-10-11 11:09:51:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/follow_test.py
2016-10-11 11:09:53:No exception in execute_script method.
2016-10-11 11:09:53:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/like_test.py
2016-10-11 11:09:54:No exception in execute_script method.
2016-10-11 11:09:54:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/login_test.py
2016-10-11 11:09:54:No exception in execute_script method.
2016-10-11 11:09:54:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/my_collect_posts_test.py
2016-10-11 11:09:54:No exception in execute_script method.
2016-10-11 11:09:54:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/post_info_test.py
2016-10-11 11:09:55:No exception in execute_script method.
2016-10-11 11:09:55:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/report_test.py
2016-10-11 11:09:55:No exception in execute_script method.
2016-10-11 11:09:55:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/search_users_test.py
2016-10-11 11:09:56:No exception in execute_script method.
2016-10-11 11:09:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/send_post_test.py
2016-10-11 11:09:56:No exception in execute_script method.
2016-10-11 11:09:56:execute_script: /Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/upload_images_test.py
2016-10-11 11:09:57:No exception in execute_script method.
